### PR TITLE
remove 'other' editor

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

File: packages/protocol/src/data/inscription-type.ts [hunk 1]
```diff
@@ -21,15 +21,3 @@ export namespace ActivityEditorType {
   export type Bpmn = 'GenericBpmnElement' | 'UserBpmnElement' | 'ManualBpmnElement' | 'ScriptBpmnElement' | 'ServiceBpmnElement' | 'RuleBpmnElement' | 'SendBpmnElement' | 'ReceiveBpmnElement';
   export type All = General | Interface | Bpmn;
 }
-
-export type OtherEditorType = 'ProcessAnnotation' | 'Unknown';
-
-export type InscriptionEditorType = EventEditorType.All | GatewayEditorType | ActivityEditorType.All | OtherEditorType;
-
-export interface InscriptionTypeBeta {
```
Comment (ivy-rew) [inline packages/protocol/src/data/inscription-type.ts:29]: the idea of this PR is mainly, to get rid of this re-declaration : as the inscriptionType with its valid element-Types already exists out of the generated inscription.ts protocol

File: packages/editor/src/context/useEditorContext.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-  type: InscriptionTypeBeta;
+  type: InscriptionType;
 }
 
 export const DEFAULT_EDITOR_CONTEXT: EditorContext = {
   pid: '',
   readonly: false,
   type: {
-    id: 'Unknown',
+    id: 'Script',
```
Comment (ivy-lli) [inline packages/editor/src/context/useEditorContext.ts:14]: Why this change? The default editor context type shouldn't be script right?

Comment (ivy-lli) [reply packages/editor/src/context/useEditorContext.ts:14]: Ah this doesn't exist in the schema right? :see_no_evil:

File: integrations/standalone/src/mock.tsx [hunk 3]
```diff
@@ -5,12 +5,12 @@ import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
...
 import ReactDOM from 'react-dom/client';
 import { URLParams } from './url-helper';
-import { InscriptionEditorType } from '@axonivy/inscription-protocol';
+import { InscriptionType } from '@axonivy/inscription-protocol';
 
 export async function start(): Promise<void> {
   const theme = URLParams.themeMode();
   const readonly = URLParams.parameter('readonly') ? true : false;
-  const type = (URLParams.parameter('type') as InscriptionEditorType) ?? undefined;
+  const type = (URLParams.parameter('type') as InscriptionType["id"]) ?? undefined;
```
Comment (ivy-lli) [inline integrations/standalone/src/mock.tsx:13]: don't know what this "id" is, but we normally use single quotes in js. Also something which prettier should fix I think?

File: packages/editor/src/components/editors/others/other-editors.tsx [hunk 4]
```diff
@@ -1,8 +0,0 @@
-import { IvyIcons } from '@axonivy/editor-icons';
-import { OtherEditorType } from '@axonivy/inscription-protocol';
-import { ReactNode } from 'react';
-import NameEditor from '../NameEditor';
-
-export const otherEditors = new Map<OtherEditorType, ReactNode>([
-  ['ProcessAnnotation', <NameEditor icon={IvyIcons.Note} hideTags={true} />]
```
Comment (ivy-lli) [inline packages/editor/src/components/editors/others/other-editors.tsx:7]: now we lose the ability to configure process annotations?

File: integrations/standalone/src/mock.tsx (hunk 3 above)
Comment (ivy-rew) [reply integrations/standalone/src/mock.tsx:13]: this is the enumeration that lists all available 'element-types'. VS-Code helped me to write that 'id' quote, gonna try again with prettier

**Review by ivy-lli — APPROVED** [review]
